src/components/Video/Video.module.scss

```diff
@@ -1,14 +1,23 @@
 .VideoContainer {
+  position: relative;
   display: flex;
   flex-direction: column;
   justify-content: center;
 
+  > * {
+    border-radius: inherit;
+  }
+
   .Controls {
+    position: absolute;
     display: flex;
     justify-content: space-between;
+    bottom: 0;
+    box-sizing: border-box;
+    width: 100%;
     padding: 0.5rem;
-    border-radius: 0 0 0.5rem 0.5rem;
-    background-color: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 1));
+    border-radius: 0 0 1rem 1rem;
```

Comment:
If inherit is used here each border side has a radius of 16px but we only require bottom-left and bottom-right